apps/miranum-bpmn-modeler-webview/src/main.ts

```diff
@@ -1,31 +1,27 @@
-import {
-    ContentController,
-    ImportWarning,
-    instanceOfModelerData,
-    setFormKeys,
-} from "./app";
+import { ContentController, ImportWarning, instanceOfModelerData, setFormKeys } from "./app";
 import { debounce, reverse, uniqBy } from "lodash";
-import {
-    FolderContent,
-    MessageType,
-    StateController,
-} from "@miranum-ide/vscode/miranum-vscode-webview";
-import { ModelerData } from "@miranum-ide/vscode/shared/miranum-modeler";
-
-// bpmn-js
-import BpmnModeler, { ErrorArray, WarningArray } from "bpmn-js/lib/Modeler";
+import { FolderContent, MessageType, StateController } from "@miranum-ide/vscode/miranum-vscode-webview";
+import { ExecutionPlatformVersion, ModelerData } from "@miranum-ide/vscode/shared/miranum-modeler";
+
 import {
     BpmnPropertiesPanelModule,
     BpmnPropertiesProviderModule,
     CamundaPlatformPropertiesProviderModule,
     ElementTemplatesPropertiesProviderModule,
+    ZeebePropertiesProviderModule,
 } from "bpmn-js-properties-panel";
 import CamundaPlatformBehaviors from "camunda-bpmn-js-behaviors/lib/camunda-platform";
-import camundaModdleDescriptors from "camunda-bpmn-moddle/resources/camunda.json";
 import ElementTemplateChooserModule from "@bpmn-io/element-template-chooser";
 import miragonProviderModule from "./app/PropertieProvider/provider";
 import TokenSimulationModule from "bpmn-js-token-simulation";
 
+// Camunda 7 descriptor
+import camundaModdleDescriptors from "camunda-bpmn-moddle/resources/camunda.json";
+// Camunda 8 descriptor
+import ZeebeBpmnModdle from "zeebe-bpmn-moddle/resources/zeebe.json";
+
+import BpmnModeler from "camunda-bpmn-js/lib/base/Modeler";
```

Comment:
Is there a particular reason why you only use the BaseModeler? Because if you use the other modelers, many of the *additionalModules* are already integrated.

Reply:
Ok didn't know that. I've refactored it!
Do you have an idea why the element templates do not get picked up for camunda 8 tho?

Reply:
What do you mean by 'not picked up'? If you have a project with element templates you have to define in the **miranum.json** where they are. Or you just put them in a folder called *element-templates*. Otherwise the extension does not know where to find them.

Reply:
When having a bpmn7 and a bpmn8 and there is an element-template folder inside the directory, then the c7 diagram shows the suggestion to use the element template but c8 does not.

Reply:
For which platform was the element templated created? Do you see a error message in the log (Click on the bug in the top right corner)?